Makefile

```diff
@@ -1,10 +1,23 @@
+# set your dissertation main file here
 DISS     = dissertation.tex
 
+# automatic main file detection
+NUMMAINS = $(shell grep --files-with-matches --exclude="_*_.tex" \
+	     '^\\documentclass' *.tex | wc -l)
+ifeq ($(NUMMAINS), 1)
+DISS     = $(shell grep --files-with-matches --exclude="_*_.tex" \
+	     '^\\documentclass' *.tex)
+else
+$(warning *** Could not determine main LaTeX file, trying to use $(DISS))
+$(warning *** - either delete all files containing '\documentclass')
+$(warning *** - or specify your main file in the Makefile)
+endif
+
 TEX      = latexmk
 TEXFLAGS = -recorder -pdf
 TEXCLEAN = -bibtex -c
 
-DOCKER   = docker run -it --rm -v $(shell pwd):/diss \
```

Comment:
Das Dockerfile hat /diss hardcoded.

Reply:
I know, that's why I added the -w $(pwd) switch to the docker run command, which acts like a WORKDIR HOSTPWD in the Dockerfile when building the container.

The main problem is that files like the synctex.gz uses the paths of the docker container (e.g. "Input:1:/diss/./dissertation.tex"), which might render the synctex file useless for the host system.

Reply:
Okay sorry, my question was more like do we still need the workdir assignment in the Dockerfile if it's added here explicitly? Only ob mobile right so can't check. If not we it's obsolete there  and can be deleted, right?

Reply:
Yep, imho we don't need the Workdir assignment in Dockerfile. I removed it in the latest commit, but haven't tested it.